File: score.cpp
#include <json/value.h>
#include <fstream>

void getQuestion(Json::Value questions, int questionNum) {
	int idx = 0;
	while (questions["riddle"]) {
		
	}
}

std::ifstream jsonData("riddles.json", std::ifstream::binary);
Json::Value questions;
jsonData >> questions;

File: score.php
<?php
	$playerScore = 0;

	$path = 'riddles.json';
	$jsonString = file_get_contents($path);
	$jsonData = json_decode($jsonString, true);
	var_dump($jsonData);

	function getQuestion($questionNum) {
		$idx = 0;
		while ($jsonData->questionNum == $questionNum) {
			
		}
	}

	function addPoints($playerScore, $questionNum) {
		return $playerScore + $points[$questionNum];
	}


?>